### DISCUSSION TITLE
Reduced storage retention, how long until Prom cleans up?

### DISCUSSION THREAD

**Discussion by sheppduck** [prometheus/prometheus#11577] [General]
Hi!

We have been hitting our storage alerts and we reduced the storage retention time today.  We assume Prom will groom off the data that now exceeds the new configuration/retention time.  If so, how long would this take (let's assume 300GB of data)?  
If its not going to actively groom off the data that is outside the retention period, how can we do this (let's say we want to groom off the last 60d)?
thanks!

**Comment by roidelapluie** [comment]
Prometheus should check this every minute.
